en/components/column-chart.md

```diff
@@ -0,0 +1,52 @@
+---
+
+# Column Chart
+
+ Category Charts are often combined to create beautiful dashboards for related sets of data. Column Charts are among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by vertical bars of equal width and differing heights. They are ideal for showing variations in the value of an item over time. Data is represented using a collection of rectangles that extend from the bottom to top of the chart towards the values of data points.
+
+
+## Column Chart Demo
+
+<img class="responsive-img" src="../images/column_chart_demo.png" srcset="../images/column_chart_demo@2x.png 2x" />
+
+The Column Chart comprises of Title and Chart area, where you can choose from Idle or Hover states.
+
+## Title
+
+The Column Chart has a title that can be changed or hidden if not needed.
+
+## Tooltip
+
+The Column Chart comes with an override for the tooltip visibility: **Tooltip Off** hides it and Tooltip On shows it on top of the series.
+
+<img class="responsive-img" src="../images/column_chart_tooltip-off.png" srcset="../images/column_chart_tooltip-off@2x.png 2x" />
+<img class="responsive-img" src="../images/column_chart_tooltip-on.png" srcset="../images/column_chart_tooltip-on@2x.png 2x" />
+
+## Styling
+
+Columns can be hidden, by setting individual columns to "No Symbol" in order to change their number, as well as hiding the Y axis and it's title. Тhe color of the series can also be changed. The annotations can be removed by deleting their content.
+
+<img class="responsive-img" src="../images/column_chart_styling.png" srcset="../images/column_chart_styling@2x.png 2x" />
+
+## Usage
+
+Use the Column Chart when you want to see the amount of change over a period of time. Always start you numeric Axis at 0, so you don't misrepresent the data. Don't use different colors for the same category. If you use time to represent the change in the category you should always set it on the horizontal axis. If you need a detailed Time-Series analysis  - consider a Line Chart with a Time-Series for this type of data.
+
+| Do                                                                                             | Don't                                                                                              |
+| ---------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| <img class="responsive-img" src="../images/column_chart_do1.png" srcset="../images/column_chart_do1@2x.png 2x" /> | <img class="responsive-img" src="../images/column_chart_dont1.png" srcset="../images/column_chart_dont1@2x.png 2x" /> |
```

Comment:
See the do - don't format we use in other topics with green and red element in the top left and use it here. I also think that based on what you wrote in the paragraph above you can add at least one more visual with do and don't.